bin/build-release-ccd-definition.sh

```diff
@@ -6,18 +6,18 @@ environment=${1:-prod}
 excludeNonProdFiles=${2:-true}
 
 if [ ${environment} == preview ]; then
-   excludedFilenamePatterns="-e *-prod.json,*HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*-GAR2GAspec-nonprod.json"
+   excludedFilenamePatterns="-e *-prod.json,*HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*-GAR2GAspec-nonprod.json,*-shuttered.json"
    #for testing HNL uplifting, comment the above line and uncomment below
    #excludedFilenamePatterns="-e *-prod.json,*-SDO-nonprod.json,*-base-nonprod.json"
    #for testing ga enhancements please remove *-GAR2GAspec-nonprod.json which are not required for release 1
 elif [ ${environment} == demo ]; then
-  excludedFilenamePatterns="-e UserProfile.json,*-prod.json,*HNL-nonprod.json,*-GAR2GAspec-nonprod.json,*CUI-nonprod.json"
+  excludedFilenamePatterns="-e UserProfile.json,*-prod.json,*HNL-nonprod.json,*-GAR2GAspec-nonprod.json,*CUI-nonprod.json,*-shuttered.json"
 elif [ ${environment} == local ]; then
-  excludedFilenamePatterns="-e *-prod.json"
+  excludedFilenamePatterns="-e *-prod.json,*-shuttered.json"
 elif [ ${excludeNonProdFiles} == true ]; then
```

Comment:
I suspect this will get applied to both PROD and AAT. The reason is the default value for excludeNonProdFiles is true. Looking at Jenkinsfile_CNP, we never set this value which means this always will be set to true and will be applicable to both PROD & AAT

Reply:
Ticket has been raised for these changes https://tools.hmcts.net/jira/browse/CIV-6870